common/SmokeTests/SmokeTest/SmokeTest.csproj

```diff
@@ -5,26 +5,23 @@
     <LangVersion>latest</LangVersion>
     <StartupObject>SmokeTest.Program</StartupObject>
   </PropertyGroup>
-
   <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
-
   <ItemGroup>
-    <PackageReference Include="Azure.Core" Version="1.4.0-dev.*" />
-    <PackageReference Include="Azure.Identity" Version="1.2.0-dev.*" />
-    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.1.0-dev.*" />
-    <PackageReference Include="Azure.Messaging.EventHubs.Processor" Version="5.1.0-dev.*" />
-    <PackageReference Include="Azure.Security.Keyvault.Secrets" Version="4.1.0-dev.*" />
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.5.0-dev.*" />
-    <PackageReference Include="Microsoft.Azure.Amqp" Version="2.4.3" />
+    <PackageReference Include="Azure.Core" Version="1.7.0" />
+    <PackageReference Include="Azure.Identity" Version="1.4.0-beta.1" />
+    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.3.0-beta.4" DoNotUpdate="" />
```

Comment:
Perhaps we should have a xml comment or a comment in the new attribute explaining why it is this way.